editions/tw5.com/tiddlers/widgets/BrowseWidget.tid

```diff
@@ -11,7 +11,7 @@ The browse widget displays an HTML file browser button that allows the user to c
 
 ! Content and Attributes
 
-The content of the `<$browse>` widget is ignored.
+The content of the `<$browse/>` widget is ignored.
```

Comment:
A lot of inconsistency has crept in, but for textual references to widgets within their own definition should use the `<<.wid>>` macro:

```
The content of the <<.wid BrowseWidget>> widget is ignored
```

(References to widgets that should be linked use the `<<.wlink>>` macro).

Reply:
Noted: 
How can I see if you made this change and accepted it?
If not how do I go about doing this?
* Cancel and resubmit?

Reply:
Are you asking how to make the requested changes? It should work if you go to https://github.com/AnthonyMuscio/TiddlyWiki5/blob/patch-24/editions/tw5.com/tiddlers/widgets/BrowseWidget.tid and click the edit button and make the changes.

I haven't edited files via the github website before, but I expect making the change in that way will cause a new commit to the same branch (patch-24). Since that branch is attached to this pull request, it will appear here.